tests/unit/test_ndproxy_table.cpp

```diff
@@ -28,7 +28,7 @@
 
 #include "openthread-core-config.h"
 
-#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
+#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_DUA_NDPROXYING_ENABLE
```

Comment:
update the marco for multicast listener table test as well?
https://github.com/openthread/openthread/blob/524d8fbd6353b35f406ab199aaad011541b96760/tests/unit/test_multicast_listeners_table.cpp#L31